pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -300,20 +322,28 @@ func (p *AnacondaInstallerISOTree) serialize() osbuild.Pipeline {
 			images,
 			manifests))
 
-		kickstartOptions, err := osbuild.NewKickstartStageOptionsWithOSTreeContainer(
-			p.KSPath,
-			p.Users,
-			p.Groups,
-			path.Join("/run/install/repo", p.PayloadPath),
-			"oci",
-			"",
-			"")
+		kickstartFile, err := fsnode.NewFile(p.KSPath, nil, nil, nil, []byte(`
+ostreecontainer --url=/run/install/repo/container --transport=oci --no-signature-verification
+rootpw --lock
+user --name fedora --groups wheel
+lang en_US.UTF-8
+keyboard us
+timezone UTC
+clearpart --all
+part /boot/efi --fstype=efi --size=512 --fsoptions="umask=0077"
+part /boot --fstype=ext2 --size=1024 --label=boot
+part swap --fstype=swap --size=1024
+part / --fstype=ext4 --grow
```

Comment:
The PR mentioned in the commit message has been merged now.  Do we have time to do this properly or should we keep this for a bit?

Reply:
We decided to do that bit in a followup.